app/models/election.js

```diff
@@ -43,6 +45,17 @@ const electionSchema = new Schema({
       ref: 'Vote',
     },
   ],
+  type: {
+    type: String,
+    enum: [ElectionTypes.NORMAL, ElectionTypes.STV],
+    default: ElectionTypes.NORMAL,
+    validate: {
+      validator: function (v) {
+        return v === ElectionTypes.NORMAL && this.seats !== 1 ? false : true;
```

Comment:
```suggestion
        return v === ElectionTypes.NORMAL && this.seats === 1;
```

Reply:
The same goes for line 64.

Reply:
I gotta break it to you, that's not the same. Back to [diskmat](https://wiki.math.ntnu.no/tma4140/2021h/start)

Reply:
But I agree that this is very confusing, and I bet there is a better way to convey the point of these two validators.